.gitignore

```diff
@@ -0,0 +1 @@
+config.yml
```

Comment:
I might delete it in the future (probably move to an explicit sample file), but as of now I prefer to keep it.